starlite/openapi/controller.py

```diff
@@ -65,6 +66,35 @@ def get_schema_from_request(request: Request) -> "OpenAPI":
             raise ImproperlyConfiguredException("Starlite has not been instantiated with OpenAPIConfig")
         return request.app.openapi_schema
 
+    def should_serve_endpoint(self, request: Request) -> bool:
+        """This method verifies that the requested path is within the allowed
+        endpoints in the openapi_config.
+
+        Args:
+            request: A [Starlite][starlite.connection.Request] instance.
+
+        Returns:
+            A boolean.
+
+        Raises:
+            [ImproperlyConfiguredException][starlite.exceptions.ImproperlyConfiguredException]: If the application
+                `openapi_config` attribute is `None`.
```

Comment:
```suggestion
            ImproperlyConfiguredException: If the application `openapi_config` attribute is `None`.
```

No need to cross-ref the type, it will get resolved as a cross ref without it, e.g:
Before:

![image](https://user-images.githubusercontent.com/20659309/187310202-46dbcdbe-1116-4da6-b7f0-f8729b1262f9.png)

after

![image](https://user-images.githubusercontent.com/20659309/187310310-2117cf39-e648-40b9-a2db-120a8f575bca.png)